spec/active_model_spec.rb

```diff
@@ -30,7 +30,7 @@
 
   describe 'Except Amex and Maestro brand' do
     it 'should reject all other valid numbers' do
-      VALID_NUMBERS.except(:amex, :maestro).each do |_, numbers|
+      VALID_NUMBERS.except(:amex, :maestro, :vr_refeicao).each do |_, numbers|
```

Comment:
describe 'Except Amex and Maestro brand' now is not reflecting real test,  also there is new describe block below
describe 'Only Amex and Mestro brands' that is testing opposite behaviour of this describe block  

maybe it is better just to remove this change